obtrusive.txt

```diff
@@ -183,6 +183,7 @@ hintaseuranta.fi###cookies-info-banner
 horoscope.fr###useCookiesCont
 hotelgranvia.com###lc_cookies-main
 hp.com###cookie_privacy_holder
+huffingtonpost.de##[class$="overlay-hp"]
```

Comment:
This is not related to cookies, I get this message to become a Facebook fan. Those kind of overlays are not the scope for this list.